mne/io/egi/tests/test_egi.py

```diff
@@ -288,4 +291,40 @@ def test_io_egi_crop_no_preload():
     assert_allclose(raw._data, raw_preload._data)
 
 
+@requires_version('mffpy', '0.5')
```

Comment:
Technically we need `mffpy v0.5.5` for this to run, but `mne.utils.requires_version` can only check against the "loose version" (e.g. 0.5). Should we edit this function to be able to check against the full v0.5.5?

Reply:
I don't think this is a problem with `requires_version`, which just uses `check_version`, which just uses `LooseVersion` from `distutils.version`. I think it's a problem with the `mffpy` 0.5.5 reporting its version as 0.5.0:
```
$ pip install mffpy
Defaulting to user installation because normal site-packages is not writeable
Collecting mffpy
  Downloading mffpy-0.5.5-py3-none-any.whl (140 kB)
     |████████████████████████████████| 140 kB 3.1 MB/s 
...
larsoner@bunk:~$ python
>>> import mne
>>> mne.utils.check_version('mffpy', '0.5')
True
>>> mne.utils.check_version('mffpy', '0.5.5')
False
>>> mne.utils.check_version('mffpy', '0.5.4')
False
>>> import mffpy
>>> mffpy.__version__
'0.5.0'
```

Reply:
And just to show how it works if the version of a lib is correct:
```
>>> import pytest
>>> pytest.__version__
'6.1.1'
>>> mne.utils.check_version('pytest', '6.1.1')
True
>>> mne.utils.check_version('pytest', '6.1.2')
False
```

Reply:
Ah good call. I'll look into why `mffpy` is reporting the wrong version.

Reply:
I figured out the issue on the `mffpy` side. Just need to merge that PR with the changes and then `mffpy` should be displaying the right version.

Reply:
And probably release a 0.5.6?

Reply:
Yes, that's the plan.